deployment.rb

```diff
@@ -118,6 +118,7 @@ def load_configuration
     'pusher_app_id'               => 'fake_app_id',
     'pusher_application_key'      => 'fake_application_key',
     'pusher_application_secret'   => 'fake_application_secret',
+    'stub_school_data'            => [:adhoc, :test].include?(rack_env),
```

Comment:
IMHO the only place we don't want these is in adhoc and test because they reseed frequently. developers and staging can pay the 4-minute cost once and have the benefit of the full data set.